app/views/users/new.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<h1>Create new user</h1>
+<%= form_with model: @user do |form| %>
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+    
+    <%= form.label :username %>
+    <%= form.text_field :username, placeholder: "Enter your username", class:"placeholder" %>
+    <%= form.label :email %>
+    <%= form.email_field :email, placeholder: "Enter your email address", class:"placeholder" %>
+    <%= form.label :password %>
+    <%= form.password_field :password, placeholder: "Enter your password", class:"placeholder" %>
+    <%= form.submit 'Submit' %>
+<% end %>
```

Comment:
- [ ] Kindly you should create a regular HTML form and comment it out. This is a requirement for this project since it will serve as a reminder of how we create forms in HTML. ⬇️ 
 
![user_form1](https://user-images.githubusercontent.com/32361963/113395466-904ecb00-93a2-11eb-8cae-8d06863d563e.png)

![user_form_tag2](https://user-images.githubusercontent.com/32361963/113395491-9b096000-93a2-11eb-872e-dbad0b071543.png)